src/librbd/Journal.h

```diff
@@ -105,6 +105,10 @@ class Journal {
   static int get_tag_owner(ImageCtxT *image_ctx, std::string *mirror_uuid);
   static int get_tag_owner(librados::IoCtx& io_ctx, std::string& image_id,
                            std::string *mirror_uuid);
+  static void is_tag_owner(ImageCtxT *image_ctx, ContextWQ *work_queue,
```

Comment:
I can't extract it inside `Journal<I>`, because in test_mock I->op_work_queue is MockContextWQ and build fails:
```
error: no matching function for call to ‘journal::Journaler::Journaler(librbd::MockContextWQ*&, SafeTimer*&, Mutex*&, librados::IoCtx&, std::string&, const string&, <brace-enclosed initializer list>)’
```